static/js/flow/emailTypes.js

```diff
@@ -31,5 +29,5 @@ export type EmailConfig = {
   title: string,
   renderSubheading: (subheading: string) => React$Element<*>,
   emailOpenParams: (args: any) => Object,
-  emailSendAction: (emailState: EmailState) => Dispatcher<EmailSendResponse>
+  emailSendParams: (emailState: EmailState) => Array<any>
```

Comment:
We should avoid using the `any` type as much as possible. What type or types should this be?